src/sentry/integrations/jira_server/integration.py

```diff
@@ -86,7 +85,6 @@ class InstallationForm(forms.Form):
         label=_("Jira URL"),
         help_text=_("The base URL for your Jira Server instance, including the host and protocol."),
         widget=forms.TextInput(attrs={"placeholder": "https://jira.example.com"}),
-        validators=[URLValidator()],
```

Comment:
Is there a more lax validator we can use? No validation will let a few classes of human error to be handled only when requests are made.

Reply:
[https://github.com/getsentry/sentry/blob/95767d455b8004ec4b4c5026d84b64b6348e6d37/src/sentry/plugins/validators/url.py] (src/sentry/plugins/validators/url.py) can probably be used. I'll update the PR and add it to the other integrations as well

Reply:
@pierredup any updates on this?